### PR TITLE
fix "dangerous-default-value" pylint problem where it is relevant 

### PR DESCRIPTION
**Pull request by naushniki**
fix "dangerous-default-value" pylint problem where it is relevant
More info:
https://pylint.pycqa.org/en/latest/user_guide/messages/warning/dangerous-default-value.html

### CODE REVIEW

**Comment by naushniki** [comment]
https://jenkins.arangodb.biz/view/RTA/job/Test-3.11.Windows/67/

https://jenkins.arangodb.biz/view/RTA/job/Upgrade-3.10-to-3.11.Linux/146/

**Comment by naushniki** [comment]
https://jenkins.arangodb.biz/view/RTA/job/Upgrade-3.10-to-3.11.Linux/148/

https://jenkins.arangodb.biz/view/RTA/job/Test-3.11.Windows/68/

File: release_tester/arangodb/sh.py [hunk 1]
```diff
@@ -273,13 +273,17 @@ def hotbackup_wait_for_ready_after_restore(self):
...
-        args=[],
-        moreargs=[],
+        args=None,
+        moreargs=None,
         result_line_handler=default_line_result,
         deadline=1000,
         progressive_timeout=100,
     ):
-        # pylint: disable=too-many-arguments disable=too-many-instance-attributes disable=dangerous-default-value
+        if args is None:
```
Comment (dothebart) [inline release_tester/arangodb/sh.py:282]: move this behind the """ line 287

**Review by dothebart — APPROVED** [review]
please fix this one sequence; else lgtm.

**Comment by naushniki** [comment]
https://jenkins.arangodb.biz/view/RTA/job/Upgrade-3.10-to-3.11.Linux/151/

**Pull request merged**